share/admin.py

```diff
@@ -227,14 +241,22 @@ def end_date_(self, obj):
         return obj.end_date.date().isoformat()
 
     def status_(self, obj):
-        if obj.status == HarvestLog.STATUS.created and (timezone.now() - obj.date_modified) > datetime.timedelta(minutes=60):
-            return '<span style="font-weight: bold;">Lost</span>'
+        if obj.status == HarvestLog.STATUS.created and (timezone.now() - obj.date_modified) > datetime.timedelta(days=1, hours=6):
+            return format_html('<span style="font-weight: bold;">Lost</span>')
         return format_html(
             '<span style="font-weight: bold; color: {}">{}</span>',
-            ['blue', 'cyan', 'red', 'green', 'goldenrod', 'orange'][obj.status],
+            self.STATUS_COLORS[obj.status_identifier],
             HarvestLog.STATUS[obj.status].title(),
         )
 
+    def restart_tasks(self, request, queryset):
+        for log in queryset.select_related('source_config'):
+            HarvesterTask().apply_async((1, log.source_config.label), {
```

Comment:
Should this have `superfluous`? Or do you see this button just for restarting failed/lost/etc. tasks